NVIDIA-Driver/install.bash

```diff
@@ -74,15 +136,11 @@ if ! sudo sh "$INSTALLER" \
     --x-library-path=/opt/nvidia/lib64 \
     --x-sysconfig-path=/etc/X11/xorg.conf.d \
     --documentation-prefix=/opt/nvidia \
-    --application-profile-path=/etc/nvidia/nvidia-application-profiles-rc.d \
+    --application-profile-path=/etc/nvidia \
```

Comment:
Why?

Reply:
Must have been an error while editing, didn't intend it

Reply:
Please delete this